lib/src/darwin.dart

```diff
@@ -9,11 +9,11 @@ class AVAudioSession {
 
   final _interruptionNotificationSubject =
       PublishSubject<AVAudioSessionInterruptionNotification>();
-  //final _routeChangeSubject = PublishSubject<AVAudioSessionRouteChangeReason>();
-  //final _silenceSecondaryAudioHintSubject =
-  //    PublishSubject<AVAudioSessionSilenceSecondaryAudioHintType>();
-  //final _mediaServicesWereLostSubject = PublishSubject<void>();
-  //final _mediaServicesWereResetSubject = PublishSubject<void>();
+  final _routeChangeSubject = PublishSubject<AVAudioSessionRouteChangeReason>();
```

Comment:
The stream type probably needs to be a new class that wraps around the reason + description.

Reply:
New class here